### PR TITLE
Cu 9cra6a file renaming

### PR DESCRIPTION
**Pull request by Hamza-A**
## Description
This PR aims to rename a couple of files for more consistency in our project

- I renamed `<Account />` to `<AccountOverview />`
- I renamed `<Logins />` to `<LoginsOverview />`
- I renamed `<ShowIdentityPage />` to `<IdentityOverviewPage />`
- I renamed `<ShowIdentity />` to `<IdentityOverview />`
- I renamed `<IdentityValidationPage />` to `<ValidateIdentityPage />`
- I renamed `<VerifyIdentity />` to `<ValidateIdentity />`
- I renamed `<IdentityValidationForm />` to `<ValidateIdentityForm />`
- I moved `<Container />`, `<H1 />`, `<H2 />`, `<NavigationBreadcrumbs />` from display component to page (except for `<LocalePage />`)
- I added spaces in all stories' title
## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## How Has This Been Tested?
There were no need to add or modify tests

## Checklist:

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: src/components/display/fewlines/AccountOverview/AccountOverview.tsx [hunk 1]
```diff
@@ -2,19 +2,14 @@ import Link from "next/link";
...
 import { NeutralLink } from "../NeutralLink/NeutralLink";
 import { ShadowBox } from "../ShadowBox/ShadowBox";
 
-const Account: React.FC = () => {
+const AccountOverview: React.FC = () => {
   return (
-    <Container>
-      <H1>Welcome to your account</H1>
-      <H2>First name last name</H2>
+    <>
```
Comment (Fuzznimp) [inline src/components/display/fewlines/AccountOverview/AccountOverview.tsx:12]: You don't need the fragment.

File: src/components/display/fewlines/AddIdentityForm/AddIdentityForm.stories.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { IdentityTypes } from "@lib/@types";
 import { AddIdentity } from "@src/components/business/AddIdentity";
 
 export default {
-  title: "pages/AddIdentityForm",
+  title: "pages/Add Identity Form",
```
Comment (Fuzznimp) [inline src/components/display/fewlines/AddIdentityForm/AddIdentityForm.stories.tsx:9]: 👍

File: src/pages/account/logins/index.tsx [hunk 3]
```diff
@@ -20,7 +24,16 @@ type LoginsProps = {
 };
 
 const LoginsPage: React.FC<LoginsProps> = ({ sortedIdentities }) => {
-  return <Logins sortedIdentities={sortedIdentities} />;
+  return (
+    <Container>
+      <Head>
+        <title>Connect Logins</title>
+      </Head>
```
Comment (Fuzznimp) [inline src/pages/account/logins/index.tsx:31]: Can you remove the head component? In my PR, I added it right into the `_document` component.

File: tests/pages/ValidateIdentityPage.test.tsx [hunk 4]
```diff
@@ -8,16 +8,15 @@ import { IdentityTypes } from "@lib/@types/Identity";
 import AlertBar from "@src/components/display/fewlines/AlertBar/AlertBar";
 import { Button } from "@src/components/display/fewlines/Button/Button";
 import { Form } from "@src/components/display/fewlines/Form/Form";
-import IdentityValidationForm from "@src/components/display/fewlines/IdentityValidationForm/IdentityValidationForm";
 import { Input } from "@src/components/display/fewlines/Input/Input";
 import {
   NavigationBreadcrumbs,
   Breadcrumbs,
 } from "@src/components/display/fewlines/NavigationBreadcrumbs/NavigationBreadcrumbs";
+import ValidateIdentityForm from "@src/components/display/fewlines/ValidateIdentityForm/ValidateIdentityForm";
```
Comment (Fuzznimp) [inline tests/pages/ValidateIdentityPage.test.tsx:16]: Hmmm, this should be 
```suggestion
import { ValidateIdentityForm } from "@src/components/display/fewlines/ValidateIdentityForm/ValidateIdentityForm";
```

**Review by Fuzznimp — CHANGES_REQUESTED** [review]

File: src/components/display/fewlines/AccountOverview/AccountOverview.tsx (hunk 1 above)
Comment (Hamza-A) [reply src/components/display/fewlines/AccountOverview/AccountOverview.tsx:12]: I do, in the future we must add more `<ShadowBox />`

File: tests/pages/ValidateIdentityPage.test.tsx (hunk 4 above)
Comment (Hamza-A) [reply tests/pages/ValidateIdentityPage.test.tsx]: OK fixed e68cb79 I removed an extra `export default` from `src/components/display/fewlines/ValidateIdentityForm/ValidateIdentityForm`

File: src/pages/account/logins/index.tsx (hunk 3 above)
Comment (Hamza-A) [reply src/pages/account/logins/index.tsx]: OK fixed e68cb79

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**